test/parallel/test-http-outgoing-renderHeaders.js

```diff
@@ -0,0 +1,38 @@
+'use strict'
+
+const common = require('../common');
+const assert = require('assert');
+
+const outHeadersKey = require('internal/http').outHeadersKey;
+const http = require('http');
+const OutgoingMessage = http.OutgoingMessage;
+
+const outgoingMessage = new OutgoingMessage();
```

Comment:
Can you isolate these tests using block scoping. That prevents the need for `outgoingMessage`, `outgoingMessage2`, etc. It also reduces the potential for the tests interacting with each other unintentionally.